packages/terra-image/src/Image.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import './Image.scss';
+
+const propTypes = {
+  /**
+   * The source for the image which will be displayed.
+   */
+  src: PropTypes.string.isRequired,
+  /**
+   * Sets the style of the image from the following values; `default`, `rounded`, `circle`, `thumbnail`.
+   */
+  variant: PropTypes.oneOf(['default', 'rounded', 'circle', 'thumbnail']),
+  /**
+   * Sets the fluid behavior of the image, which is `nonfluid` by default.
+   */
+  isFluid: PropTypes.bool,
+  /**
+   * The text content that specifies an alternative text for an image.
+   */
+  alt: PropTypes.string.isRequired,
+  /**
+   * Sets the height of the image.
+   */
+  height: PropTypes.number,
+  /**
+   * Sets the width of the image.
+   */
+  width: PropTypes.number,
+};
+
+const defaultProps = {
+  variant: 'default',
+  isFluid: false,
+  alt: ' ',
+};
+
+const Image = ({
+  src,
+  variant,
```

Comment:
I think 'rounded', 'circle', 'thumbnail' fall inline better with how we've using variant.
Intent is used for things like primary, secondary, warning, negative, positive. I'd recommend we keep it as variant.